examples/syslog_friendly_format.rs

```diff
@@ -0,0 +1,24 @@
+use std::io::Write;
+
+fn main() {
+    match std::env::var("RUST_LOG_STYLE") {
+        Ok(s) if s == "SYSTEMD" => env_logger::builder()

```

Comment:
```suggestion
    match std::env::var_os("RUST_LOG_STYLE").as_deref() {
        Some(b"SYSTEMD") => env_logger::builder()
```

because `env::var` is lame :smile:

Reply:
Feels more natural, totally true. Would you mind changing it in a PR? :)

Reply:
I did just merge it as it seemed trivial. Should have waited :)